tests/framework/builder.py

```diff
@@ -8,18 +8,12 @@
 from pathlib import Path
 from conftest import init_microvm
 from framework.artifacts import Artifact, DiskArtifact, Snapshot, SnapshotType
+from framework.artifacts import NetIfaceConfig
```

Comment:
Nit:
If the line is not too long so the styling tests fail, "NetIfaceConfig" could be appended to the line above

Reply:
Can be added as a multi-line import too.

Reply:
Fixed by multiline.